recipe/meta.yaml

```diff
@@ -27,25 +27,20 @@ requirements:
     - toml
   run:
     - backoff
-    - dask
     - datashader
     - earthaccess >=0.5.1
     - fiona
     - geopandas
-    - geoviews
     - h5netcdf
     - h5py
     - holoviews
     - hvplot
-    - intake
-    - intake-xarray
     - matplotlib-base
     - numpy
-    - pandas
     - python >=3.0
     - requests
+    - s3fs
     - shapely
-    - tqdm
     - xarray
```

Comment:
@JessicaS11, can you double check if these dependency updates look ok? Trying to match up with what's in https://github.com/icesat2py/icepyx/blame/v0.8.0/requirements.txt

Reply:
@weiji14 Looks great except `pandas`, which is required for running the subsetting example notebook. I've re-added it. Thanks for noticing these needed updating and taking care of it!